packages/expo-image-picker/src/ImagePicker.ts

```diff
@@ -196,17 +199,53 @@ export async function launchCameraAsync(
 export async function launchImageLibraryAsync(
   options?: ImagePickerOptions
 ): Promise<ImagePickerResult> {
+  const mappedOptions = mapDeprecatedOptions(options ?? {});
+
   if (!ExponentImagePicker.launchImageLibraryAsync) {
     throw new UnavailabilityError('ImagePicker', 'launchImageLibraryAsync');
   }
-  if (options?.allowsEditing && options.allowsMultipleSelection) {
+  if (mappedOptions?.allowsEditing && mappedOptions.allowsMultipleSelection) {
     console.warn(
       '[expo-image-picker] `allowsEditing` is not supported when `allowsMultipleSelection` is enabled and will be ignored.' +
         "Disable either 'allowsEditing' or 'allowsMultipleSelection' in 'launchImageLibraryAsync' " +
         'to fix this warning.'
     );
   }
-  return await ExponentImagePicker.launchImageLibraryAsync(options ?? {});
+  return await ExponentImagePicker.launchImageLibraryAsync(mappedOptions);
```

Comment:
This might be better than an additional `mappedOptions` 🙂
```suggestion
  return await ExponentImagePicker.launchImageLibraryAsync({
    ...options,
    mediaTypes: parseMediaTypes(options.mediaTypes ?? ['images']),
  });
```

Reply:
We are using `mapDeprecatedOptions` in `launchCameraAsync` and `launchImageLibraryAsync`, so we would have to repeat the code, I think I prefer having a separate function is better.